README.md

```diff
@@ -116,16 +116,19 @@
 You will encounter the following log message in your GitHub Actions Pipelines:
 
 ```
-WARNING! Using --password via the CLI is insecure. Use --password-stdin.
+WARNING! Using -*** the CLI is insecure. Use --password-stdin.
```

Comment:
What for?

Reply:
I just copy & paste what I see from the output which is a little bit different from what you saw before.

Reply:
Could you please restore original state of this line, for it makes this message rather hard to understand.